community/scene.py

```diff
@@ -0,0 +1,61 @@
+import pyglet
+
+__WIDTH__ = 852
+__HEIGHT__ = 480
+
+def convert_hashColor_to_RGBA(color):
+	if '#' in color:
+		c = color.lstrip("#")
+		c = max(6-len(c),0)*"0" + c
+		r = int(c[:2], 16)
+		g = int(c[2:4], 16)
+		b = int(c[4:], 16)
+		color = (r,g,b,255)
+	return color
+
+class Scene(pyglet.sprite.Sprite):
+	def __init__(self, window, texture=None, color="#000000"):
+		self.window = window
+		width = window.width
+		height = window.height
+		x = width/2
+		y = height/2
+		if texture is None:
+			self.texture = pyglet.image.SolidColorImagePattern(convert_hashColor_to_RGBA(color)).create_image(width,height)
+		else:
+			self.texture = texture
```

Comment:
Texture should be tiled. Not sure how such a thing could be done easily with Pyglet, but you can set a texture to repeat in OpenGL like so:

```py
gl.glTexParameteri(gl.GL_TEXTURE_2D, gl.GL_TEXTURE_WRAP_S, gl.GL_REPEAT)
gl.glTexParameteri(gl.GL_TEXTURE_2D, gl.GL_TEXTURE_WRAP_T, gl.GL_REPEAT)
```

(Don't forget to bind the texture obviously.)